server/views/taskList/caTasksFinalChecks.pug

```diff
@@ -1,49 +1,34 @@
 extends taskListTemplate
 
 block tasks
-    if licenceStatus.decisions.bassReferralNeeded
-        div.taskListItem.borderBottomLight
-            include tasks/bassOfferTask
-    else
-        div.taskListItem.borderBottomLight
-            include tasks/curfewAddressTask
-
-    - var curfewAddressReady = licenceStatus.decisions.curfewAddressApproved
-    - var bassReady = licenceStatus.decisions.bassReferralNeeded && licenceStatus.tasks.bassAreaCheck === 'DONE' && !licenceStatus.decisions.bassWithdrawn
-    - var bassExcluded = ['Unavailable', 'Unsuitable'].includes(licenceStatus.decisions.bassAccepted)
-
-    if (curfewAddressReady || bassReady && !bassExcluded)
-        div.taskListItem.borderBottomLight
-            include tasks/riskManagementTask
-
-        div.taskListItem.borderBottomLight
-            include tasks/victimLiaisonTask
-
-        div.taskListItem.borderBottomLight
-            include tasks/curfewHoursTask
-
-        div.taskListItem.borderBottomLight
-            include tasks/additionalConditionsTask
-
-        div.taskListItem.borderBottomLight
-            include tasks/reportingInstructionsTask
-
-        div.taskListItem.borderBottom
-            include tasks/finalChecksTask
-
-        div.taskListItem.borderBottomLight
-            include tasks/postponementTask
-
-    div.taskListItem.borderBottom
-        include tasks/HDCRefusalTask
-
-    div.taskListItem.borderBottom
-        if allowedTransition === 'caToDmRefusal'
-            include tasks/caSubmitRefusalTask
-        else if allowedTransition === 'caToRo'
-            if licenceStatus.decisions.bassReferralNeeded
+    each task in tasklistModel
+        -var taskName = task.task
+        div.taskListItem.borderBottomLight
+            if taskName === 'bassOfferTask'
```

Comment:
can't you do all of this with a dynamic include i.e.
```javascript
include `tasks/${taskName}`
```

Reply:
:disappointed: 
Agreed, it doesn't, nor will it ever.  See https://github.com/pugjs/pug/issues/2030